configs/etl/reference.yaml

```diff
@@ -137,3 +137,14 @@ gwas_ingest:
     min_r2: 0.5
     k: 6.4 # Empiric constant that can be adjusted to fit the curve, 6.4 recommended.
   machine: ${machine.default}
+
+finngen_ingest:
+  inputs:
+    phenotype_table_url: https://r7.finngen.fi/api/phenos
```

Comment:
Any reason why we are not using the R8 release?

Reply:
Updated to use FinnGen release 8